ui/src/components/SecretForm/index.jsx

```diff
@@ -225,12 +225,22 @@ export default class SecretForm extends Component {
             }>
             {showSecret && (
               <ListItem className={classes.editorListItem}>
-                <CodeEditor
+                 <TextField
+                  required
+                  label="Enter secret"
+                  name="editorValue"
                   onChange={this.handleEditorChange}
                   mode="yaml"
                   lint
-                  value={editorValue}
+                  fullWidth
+                  autoFocus
                 />
+<!--              <CodeEditor
+                  onChange={this.handleEditorChange}
+                  mode="yaml"
+                  lint
+                  value={editorValue}
+                /> -->
               </ListItem>
             )}
           </List>
```

Comment:
Try removing these changes and replacing them with:

```diff
diff --git a/ui/src/components/SecretForm/index.jsx b/ui/src/components/SecretForm/index.jsx
index 8245b7127..e8a124385 100644
--- a/ui/src/components/SecretForm/index.jsx
+++ b/ui/src/components/SecretForm/index.jsx
@@ -106,9 +106,7 @@ export default class SecretForm extends Component {
       ? addYears(new Date(), 1000)
       : this.props.secret.expires,
     editorValue: this.props.isNewSecret
-      ? safeDump({
-          foo: 'bar',
-        })
+      ? ''
       : safeDump(this.props.secret.secret),
     showSecret: this.props.isNewSecret,
   };
@@ -226,6 +224,7 @@ export default class SecretForm extends Component {
             {showSecret && (
               <ListItem className={classes.editorListItem}>
                 <CodeEditor
+                  placeholder="foo:bar"
                   onChange={this.handleEditorChange}
                   mode="yaml"
                   lint
```

Does that work?

Reply:
okay, thank you.